packages/client/src/views/RecordAudit/History.tsx

```diff
@@ -246,7 +246,8 @@ export const GetHistory = ({
     role:
       isSystemInitiated(item) || !item.user?.systemRole
         ? intl.formatMessage(getSystemType(item.system?.type))
-        : item.user.role.labels.find((label) => label.lang === 'en')?.label,
+        : item.user.role.labels.find((label) => label.lang === intl.locale)
+            ?.label,
```

Comment:
Let's use this function: https://github.com/opencrvs/opencrvs-core/blob/2a4acdbc6ff01a0bceb284d5aa5fd17f57dce9e6/packages/client/src/views/SysAdmin/Config/UserRoles/utils.ts#L16 as this has a fallback

Reply:
And we can use useSelector along with https://github.com/opencrvs/opencrvs-core/blob/2a4acdbc6ff01a0bceb284d5aa5fd17f57dce9e6/packages/client/src/i18n/selectors.ts#L21 to get the current language

Reply:
Thanks for the feedbacks @Zangetsu101 . Changed accordingly.